app/models/mentee_application_validation.rb

```diff
@@ -34,9 +34,17 @@ def validate
 
   def dry_validation_step_1
     Dry::Validation.Schema do
+      configure do
+        config.messages_file = 'config/locales/dry.en.yml'
+
+        def unique?(attr_name, value)
+          MenteeApplication.where(attr_name => value).empty?
+        end
+      end
+
       required(:first_name).filled(:str?)
       required(:last_name).filled(:str?)
-      required(:email).filled(format?: EMAIL_REGEX)
+      required(:email).filled(format?: EMAIL_REGEX, unique?: :email)
       required(:gender) { filled? & (eql?('male') | eql?('female') | eql?('other'))}
```

Comment:
ah, I also would like to add ENV variable that will keep the information about actual edition we are working on